student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Student < Person
+  attr_reader :classroom
+
+  def initialize(id, classroom, name = 'Unknown', age = 0)
```

Comment:
- [ ] Since the constructor here extends parent's constructor by adding @classroom and a parameter for it, kindly remove id and the initialization to age.